### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by brianstm**
## About your game

What is your game about? 
> Get to the last level and win while avoiding all the obstacles!

How to play your game?
> Use the W and D key to control, W to go up, D to go right (there is gravity), but beware of the pipes!

### CODE REVIEW

**Comment by LucasHT22** [comment]
Please complete the checks!
## Code
Check off the items that are true.
- [ ] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [ ] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [ ] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [ ] The game runs without errors. 
- [ ] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [ ] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [ ] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [ ] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

**Comment by brianstm** [comment]
Please complete the checks!

##Code
Check off the items that are true.

- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [/games directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors.
- [x] The name of the file/game contains only alphanumeric characters, -s, or _s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)
Image (If an image is used)
- [x] The image is in the [/games/img directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. Example: sokoban_plus.js -> sokoban_plus.png.

**Comment by LucasHT22** [comment]
use setBackground() to make the background fill the entire screen!
did you use any game for inspiration?

**Comment by brianstm** [comment]
yes the original flappy bird game

**Comment by LucasHT22** [comment]
no inspiration from the Sprig itself?

**Comment by brianstm** [comment]
yes the flappy_bird_but_no_gravity_and_worse game from sprig, design i got inspiration from google

**Comment by LucasHT22** [comment]
just this one? I'm asking because afterInput has variables with names in Portuguese, it's exactly the same as my game flurffy_bird

File: flappy_duck.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+  getFirst(duck).x += 1;
+});
+
+var gravity = 1000;
+    var gravityfunc = setInterval(function(){
+    getFirst(duck).y +=1 ;
+    gravity--;
+    },500);
+
+var time = 45;
```
Comment (LucasHT22) [inline flappy_duck.js:250]: don't you think that 45 seconds is not a short time? I can't do it in this time ever haha

**Comment by brianstm** [comment]
oh my bad, i think i got some ideas from flurffy bird too.

I did it in 40 seconds, thats why i made it 45. haha.

**Comment by LucasHT22** [comment]
> 



> oh my bad, i think i got some ideas from flurffy bird too.
> 
> I did it in 40 seconds, thats why i made it 45. haha.

no problem! just add your references/inspirations at the top of the code.

wow 40s!

**Comment by brianstm** [comment]
okok sure thing.

**Comment by leomcelroy** [comment]
Nice art in this one! 

It seems you need to make a few more changes before you're ready to get your Sprig. I'm going to close this PR for now. Feel free to reopen if you want to continue working on your game.

**Pull request closed without merging**